src/RefreshManager.m

```diff
@@ -486,6 +486,7 @@ -(void)refreshFeed:(Folder *)folder fromURL:(NSURL *)url withLog:(ActivityItem *
         if (![theLastUpdateString isEqualToString:@""])
         {
             [myRequest addRequestHeader:@"If-Modified-Since" value:theLastUpdateString];
+            [myRequest addRequestHeader:@"A-IM" value:@"feed"];
```

Comment:
@Aeyoun sounds correct to me. have you got any feed I can test with? :) Thanks for the PR by the way!